starfish/test/test_utils.py

```diff
@@ -0,0 +1,65 @@
+from collections import OrderedDict
+
+import numpy as np
+
+from starfish import Codebook, ImageStack
+from starfish.types import Features, Indices
+
+
+def imagestack_with_coords_factory(stack_shape: OrderedDict, coords: OrderedDict) -> ImageStack:
```

Comment:
Also, it seems kind of weird that the same z coordinates would be assigned to multiple z layers.  It looks like the one use case of this method has it producing a single z-layer image, which avoids the issue, but if this is a requirement, it should be documented.

Reply:
ah yea I'll make the order independent for coords and imagestack shape good call. But since the physical zrange is independent from the z-layer anyway does it matter too much if it's the same? As long as it's some kind of range?